package.json

```diff
@@ -65,6 +66,10 @@
     "istanbul": "0.4.5",
     "mocha": "3.0.2",
     "npm-run-all": "3.1.0",
+    "postcss": "5.2.4",
+    "postcss-cli": "2.6.0",
+    "postcss-cssnext": "2.8.0",
+    "postcss-normalize": "0.3.0",
```

Comment:
> Moving it allows us to update normalize.css independently.

+1

I'm wondering however if we should not cut the middleman and instead rely on `"normalize.css": "3.0.1"` and copying it ourselves like we do with Font Awesome.
`postcss-normalize` seems barely maintained, is on `normalize.css` v4 (which is neither convenient for us to stay with what we currently have, or to update to v5 later). It does seem like a bottleneck.
I don't know too much about PostCSS but it seems like we can just give it `normalize.css` directly from the `node_modules` dir, no?
What do you think of that solution overall?